pyemma/msm/ui/chapman_kolmogorov.py

```diff
@@ -127,7 +131,22 @@ def chapman_kolmogorov(dtrajs, lag, K, nsets=2, sets=None):
         C=Ccc_k.toarray()
         for l in range(len(sets)):            
             w=np.zeros(len(lcc))
-            inds = lccmap.map(sets[l])
+            """There is a problem here - the initial sets will probably be bigger than the lcc at time k*\tau"""
```

Comment:
Am 09.03.2015 um 12:21 schrieb Benjamin Trendelkamp-Schroer:
> In pyemma/msm/ui/chapman_kolmogorov.py
> <https://github.com/markovmodel/PyEMMA/pull/108#discussion_r26027976>:
> 
>> @@ -127,7 +131,22 @@ def chapman_kolmogorov(dtrajs, lag, K, nsets=2, sets=None):
>>          C=Ccc_k.toarray()
>>          for l in range(len(sets)):            
>>              w=np.zeros(len(lcc))
>> -            inds = lccmap.map(sets[l])
>> +            """There is a problem here - the initial sets will probably be bigger than the lcc at time k*\tau"""
> 
> No this is 'unfortunately' just a note that this needs a bug fix
> indicating where and why the problem occurs.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/markovmodel/PyEMMA/pull/108/files#r26027976>.
> 
I'm talking of the ten lines of code below this comment, dunno how to
comment on a range of lines.